### PR TITLE
Connection Request Button

### PR DESCRIPTION
**Pull request by hs05june**
## Related Issue

#521 
## Screenshots or GIF (In case of UI changes):

[Gif of change](https://user-images.githubusercontent.com/103871275/205926818-056ebe1e-5d18-458a-9511-9618ce0c91cb.webm)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/CardTabs/Users/index.js [hunk 1]
```diff
@@ -90,8 +100,13 @@ const UserCard = (props) => {
...
-                  onClick={() => user.onClick}
+                  onClick={e => {
+                    handleOnClick(e);
+                  }}
                   data-testId={index == 0 ? "UserAdd" : ""}
+                  sx={{
+                    cursor: "pointer"
+                  }}
                 >
                   <img src={AddUser} />
```
Comment (Sarfraz-droid) [inline src/components/CardTabs/Users/index.js:111]: Add conditional in `` tag itself.
Handle the state using `useState` hook.

Currently the handle click function is adding up extra unnecessary complexity.

**Review by Sarfraz-droid — CHANGES_REQUESTED** [review]

File: src/components/CardTabs/Users/index.js (hunk 1 above)
Comment (Sarfraz-droid) [reply src/components/CardTabs/Users/index.js:111]: create a state inside the <UserElement /> instead. inside the component

Comment (Sarfraz-droid) [reply src/components/CardTabs/Users/index.js:111]: yes

**Review by Sarfraz-droid — APPROVED** [review]

**Comment by hs05june** [comment]
@Maahi10001 @Niweera Kindly review this PR. and suggest I need to make

**Review by Maahi10001 — APPROVED** [review]

**Pull request merged**